init-sensor.h

```diff
@@ -37,6 +38,7 @@ void loadSensorConfig() {
 }
 
 void initSensorType() {
+  sensorConfig.sensorType = SENSOR_INA250A2PW;
   uint8_t sensorType = sensorConfig.sensorType;
   switch (sensorType) {
```

Comment:
Why we should init uint8_t sensorType variable instead using switch (sensorConfig.sensorType)?